### PR TITLE
Create _goodbye.siml

### PR DESCRIPTION
**Pull request by jbondguy007**
May or may not conflict with !ai goodbye. I don't know what I'm doing.

### CODE REVIEW

File: Brain/ArchiBoT/_goodbye.siml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Siml>
+  <Concept Name="_goodbye" Type="Public">
+    <Model>
+      <Pattern>GOODBYE *</Pattern>
```
Comment (Thielak) [inline Brain/ArchiBoT/_goodbye.siml]: Greetings and similar things are already in AI Base/Salutations.siml
For example https://github.com/JustArchi/ArchiBoT/blob/master/Brain/ArchiBoT/AI%20Base/Salutations.siml#L277

File: Brain/ArchiBoT/_goodbye.siml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <Pattern>BYE *</Pattern>
```
Comment (Thielak) [inline Brain/ArchiBoT/_goodbye.siml]: You can only have one \<Pattern\> per \<Model\>, if you want multiple you should use multiple \<Item\> inside \<Pattern\> like you did in other places.

File: Brain/ArchiBoT/_goodbye.siml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      <Response>
+        <Random>
+          <Item>Goodbye</Item>
+          <Item>Farewell</Item>
+          <Item>Take care<Item>
+          <item>Bye</Item>
+          <item>So long</Item>
+          <item>Sayonara</Item>
+          <item>Au revoir</Item>
+          <item>Godspeed</Item>
```
Comment (Thielak) [inline Brain/ArchiBoT/_goodbye.siml]: Tags are mismatched, with \<item\> to open and \</Item\> to close.
Actually, tags in SIML are case-sensitive, so \<item\> wouldn't work at all, they require the first letter to be capitalized.

File: Brain/ArchiBoT/_goodbye.siml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      </Response>
+    </Model>
+
+    <Model>
+      <Pattern>RANDOMGOODBYE</Pattern>
+      <Response>
+        <Random>
+          <Item>Goodbye</Item>
+          <Item>Farewell</Item>
+          <Item>Take care<Item>
```
Comment (Thielak) [inline Brain/ArchiBoT/_goodbye.siml]: The ending tag is missing here, the line should be  `<Item>Take care</Item>`

File: Brain/ArchiBoT/_goodbye.siml (hunk 2 above)
Comment (Thielak) [inline Brain/ArchiBoT/_goodbye.siml]: (in this case only the first Pattern would be matched, the others wouldn't work)

File: Brain/ArchiBoT/_goodbye.siml (hunk 1 above)
Comment (jbondguy007) [inline Brain/ArchiBoT/_goodbye.siml]: Thanks for the info!

**Comment by jbondguy007** [comment]
@Thielak Am I doing it right? Updated my file.

**Comment by Thielak** [comment]
Aye aye

**Pull request merged**